pootle/apps/pootle_store/views.py

```diff
@@ -183,7 +184,7 @@ def get_step_query(request, units_queryset):
                     # (unless PostreSQL is used and `distinct(field_name)`)
                     match_queryset = match_queryset \
                         .annotate(sort_by_field=Max(max_field)) \
-                        .order_by(sort_order)
+                        .order_by(sort_order, "store__pootle_path", "pk")
```

Comment:
in mysql and postgres they will not give back a determinate order, unless we specify something unique - ie pk